back/src/models/Category.go

```diff
@@ -4,8 +4,10 @@ import "gorm.io/gorm"
 
 type Category struct {
 	gorm.Model
-	Name           string `json:"name"`
-	ThumbnailURL   string `json:"thumbnailURL"`
-	Slug           string `json:"slug"           gorm:"unique"`
-	OrderOfDisplay *uint  `json:"orderOfDisplay" gorm:"unique"`
+	Name           string    `json:"name"`
+	ThumbnailURL   string    `json:"thumbnailUrl"`
```

Comment:
Why did you modify this ? Keep it as it was

Reply:
You have created an inconsistency here : 
- https://github.com/KrispyTech/airneis/pull/78/files#diff-328223e6b0390f7ee84d18be3eccdb3e59c201aa158ca9f54d8ca2f77465edccR46
- https://github.com/KrispyTech/airneis/pull/78/files#diff-ba1d263262df9422533f8a0063995b401182fcfde22a9fe563e16243d28fc249R6